deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/ansible/ansible_inventory.py

```diff
@@ -80,6 +80,8 @@ def write_inventory_to_file(self):
             inventory.write(self.__str__())
 
     def delete_playbook_file(self):
+        """Delete all created playbook files
+        """
```

Comment:
```suggestion
        """Delete all created playbook files"""
```

Reply:
The new modification can be reviewed in the commit:[bba01140837453f908a9dafefa89308ee9a5103e](https://github.com/wazuh/wazuh-qa/commit/bba01140837453f908a9dafefa89308ee9a5103e)